python/opae.admin/opae/admin/sysfs.py

```diff
@@ -596,6 +599,26 @@ def aer(self, values):
         except CalledProcessError as err:
             self.log.warn('error setting aer: %s', err)
 
+    def reset_bridge(self):
+        """reset_bridge Reset the devices under root port bridge object.
+
+        Notes:
+            This relies on calling 'setpci' and will log an exception if an
+            error was encountered while calling 'setpci'.
+        """
+        try:
+            # Get current bridge control value
+            bc = int(call_process(self._bridge_ctrl_cmd), 16)
+            # Enable bus reset
+            bc_reset = bc | 0x40
+            call_process(f"{self._bridge_ctrl_cmd}=0x{bc_reset:x}")
+            time.sleep(0.1)
```

Comment:
There's nothing to poll as far as I know. It's just holding a reset wire to a device that's invisible. I picked them because the values are certainly long enough relative to the clocks and fast enough that no user will notice/care.

Reply:
You're thinking about them more than I did. They worked, were obviously long enough and I moved on to work on the problem I was really trying to solve.